setup.py

```diff
@@ -36,18 +36,18 @@ def read_file(filename: str) -> str:
         'flake8',
         'flake8-bugbear',
         'flask',
-        'google-api-python-client==2.10.0',
-        'google-auth==1.27.0',
-        'google-cloud-logging==3.1.1',
-        'google-cloud-pubsub==2.3.0',
-        'google-cloud-secret-manager==2.2.0',
-        'google-cloud-storage==1.25.0',
+        'google-api-python-client>=2.10.0,<3',
+        'google-auth>=1.27.0,<2',
+        'google-cloud-logging>=3.1.1,<4',
+        'google-cloud-pubsub>=2.3.0,<3',
+        'google-cloud-secret-manager>=2.2.0,<3',
+        'google-cloud-storage==1.25.*',
         'Jinja2==3.0.3',
         'kubernetes',
-        'protobuf==3.20.1',
+        'protobuf==3.20.2',
```

Comment:
Did you mean to keep the hard pin for `protobuf`?

Reply:
Same here, this comes directly from `hail`, to avoid conflicts.

Reply:
Shouldn't `pip` automatically pull the requirements that `hail` needs from `hail`'s setup.py?

Reply:
Yes, you're right! Changed.